From: Luca Beltrame <lbeltrame@kde.org>
Date: Mon, 07 Jan 2013 19:34:41 +0000
Subject: Fix unresolved symbols appearing in PyKDE4.
X-Git-Url: http://quickgit.kde.org/?p=pykde4.git&a=commitdiff&h=bdab774f1f0874d7f0a6e14d7885d73c8101b3e9
---
Fix unresolved symbols appearing in PyKDE4.

Apparently the CMake policy responsible for this is getting set back
somewhere in the build system.

Reporter, please close the bug if the issue is no longer applicable. I
can't reproduce it.

CCBUG: 312618
---


--- a/kpythonpluginfactory/CMakeLists.txt
+++ b/kpythonpluginfactory/CMakeLists.txt
@@ -4,7 +4,7 @@
     kpythonpluginfactory.cpp)
 
 GET_FILENAME_COMPONENT(LIB_PYTHON ${PYTHON_LIBRARIES} NAME)
-ADD_DEFINITIONS(-DLIB_PYTHON=\\"${LIB_PYTHON}\\")
+ADD_DEFINITIONS(-DLIB_PYTHON="${LIB_PYTHON}")
 ADD_DEFINITIONS(-DKDE_DEFAULT_DEBUG_AREA=15000)
 
 kde4_add_plugin(kpythonpluginfactory ${kpythonpluginfactory_SRCS})

